### PR TITLE
update regex validation for the env value attribute

### PR DESCRIPTION
**Pull request by ashishmax31**
Signed-off-by: ashish <asnaraya@redhat.com>

### CODE REVIEW

File: tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: mock-operator.v0.0.9
```
Comment (garrettthomaskth) [inline tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml:1]:
```suggestion
name: mock-operator-invalid-env-value.v0.0.9
```

Comment (garrettthomaskth) [reply tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml:1]: And we could maybe rename `tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml` to `tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator-invalid-env-value.v0.0.9.yml`

Comment (garrettthomaskth) [reply tests/testdata/addons/mock-operator-with-imagesets/addonimagesets/stage/mock-operator.v0.0.9.yml:1]: Or if that is not good, we could at least put a comment in these files to tell what the point of them are. It is a little confusing just having multiple versions of the addon in the test data folder without any description of what they are testing

**Comment by ashishmax31** [comment]
@garrettthomaskth Made the requested changes.

**Review by garrettthomaskth — APPROVED** [review]
lgtm!

**Pull request merged**